### PR TITLE
View In Room, episode 1

### PR DESCRIPTION
**Pull request by saolsen**
vir

Stubs out the method to call to view an image in room. Next step is implementing that function in Eigen.

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/ArtworkActions.tsx [hunk 1]
```diff
@@ -77,12 +78,14 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
             </UtilButton>
           </TouchableWithoutFeedback>
           {Constants.AREnabled && (
-            <UtilButton pr={3}>
-              <Box mr={0.5}>
-                <EyeOpenedIcon />
-              </Box>
-              <Sans size="3">View in Room</Sans>
-            </UtilButton>
+            <TouchableWithoutFeedback onPress={() => ApiModule.presentAugmentedRealityVIR("url", 1, 2, "slug", "id")}>
```
Comment (sweir27) [inline src/lib/Scenes/Artwork/Components/ArtworkActions.tsx]: Should these be passing in an actual `url`, `slug`, and `id` vs. strings?

**Comment by sweir27** [comment]
@saolsen I _think_ that peril comment is right re: incrementing the native version as well.

File: src/lib/Scenes/Artwork/Components/ArtworkActions.tsx (hunk 1 above)
Comment (ashfurrow) [reply src/lib/Scenes/Artwork/Components/ArtworkActions.tsx]: Yeah good call. Maybe we could even modify the ObjC alert to show the data that's passed into the `presentAugmentedRealityVIR` function.

Comment (ashfurrow) [reply src/lib/Scenes/Artwork/Components/ArtworkActions.tsx]: That would be doable with the [`NSString` method `stringWithFormat:`](https://developer.apple.com/documentation/foundation/nsstring/1497275-stringwithformat). Something like:

```objc
...
message: [NSString stringWithFormat:@"slug %@, width %f, height %f", slug, width, height]
...
```

Basically a C `printf`-style format string, with `%@` calling ObjC's equivalent of `toString()`.

**Pull request merged**